docs/docs/roadmap/previous-releases.md

```diff
@@ -1,12 +1,11 @@
 <!--
 order: false
+title: Previous Releases
 parent:
   title: Previous Releases
   order: 7
 -->
 
-# Previous Releases
-
```

Comment:
ditto: The title is not visible on GH.

Reply:
To keep a conherent title in menu on the left hand side, we use the title in markdown metadata, if we add it for GH, the title is duplicated as shown here.
![image](https://github.com/cosmos/gaia/assets/98309852/382886a6-8365-4f54-a8ba-83eed46ad3ac)